e2e-tests/playwright.config.ts

```diff
@@ -11,12 +11,8 @@ import { defineConfig, devices } from '@playwright/test';
 export default defineConfig({
   timeout: 80000,
   testDir: './playwright',
-  /* Run tests in files in parallel */
-  fullyParallel: true,
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
-  /* Retry on CI only */
-  retries: process.env.CI ? 2 : 0,
```

Comment:
It's better to receive a flaky test than an error test and have the build fail.
Always failing the build instead of receiving a flaky test will burden pull request approvals and create bottlenecks

Reply:
> t's better to receive a flaky test than an error test and have the build fail.
> Always failing the build instead of receiving a flaky test will burden pull request approvals and create bottlenecks

Agreed unless there is a reason why we should report a flaky test as a failure. :)

Reply:
I don't think it can create bottlenecks because it might pass when e2e pipeline is re-triggered because its just flaky test that happen to fail on 1st attempt.
I was considering marking tests as failures because it forces us to address the issues directly, rather than increasing more flaky tests. 
also It increases execution time and does not give confidence in the run when it reports some tests flaky. The majority of the time, tests are flaky due to not having proper waiting mechanisms for elements.
Also consider a scenario where actual UI or element is flaky in that case the test might pass on next retry.

But definitely it might increase  number of re-testing on PR.

I can revert back keep the retries. wdyt? @josephca @gustavolira

Reply:
Okay, So I have reverted the change.